metrics/resizer.go

```diff
@@ -41,31 +41,31 @@ func (a *loopSecondsTotalAdapter) Add(value float64) {
 }
 
 var (
-	resizerSuccessLoopTotal = prometheus.NewCounterVec(prometheus.CounterOpts{
+	resizerSuccessResizeTotal = prometheus.NewCounter(prometheus.CounterOpts{
 		Namespace: MetricsNamespace,
-		Name:      ResizerSuccessLoopTotalKey,
-		Help:      "counter that indicates how many volume expansion processing loops succeed.",
-	}, []string{"pvc_namespace", "pvc_name"})
+		Name:      ResizerSuccessResizeTotalKey,
+		Help:      "counter that indicates how many volume expansion processing resizes succeed.",
+	})
 
-	resizerFailedLoopTotal = prometheus.NewCounterVec(prometheus.CounterOpts{
+	resizerFailedResizeTotal = prometheus.NewCounter(prometheus.CounterOpts{
 		Namespace: MetricsNamespace,
-		Name:      ResizerFailedLoopTotalKey,
-		Help:      "counter that indicates how many volume expansion processing loops are failed.",
-	}, []string{"pvc_namespace", "pvc_name"})
+		Name:      ResizerFailedResizeTotalKey,
+		Help:      "counter that indicates how many volume expansion processing resizes fail.",
+	})
 
 	resizerLoopSecondsTotal = prometheus.NewCounter(prometheus.CounterOpts{
 		Namespace: MetricsNamespace,
 		Name:      ResizerLoopSecondsTotalKey,
 		Help:      "counter that indicates the sum of seconds spent on volume expansion processing loops.",
 	})
 
-	ResizerSuccessLoopTotal *resizerSuccessLoopTotalAdapter = &resizerSuccessLoopTotalAdapter{metric: resizerSuccessLoopTotal}
-	ResizerFailedLoopTotal  *resizerFailedLoopTotalAdapter  = &resizerFailedLoopTotalAdapter{metric: resizerFailedLoopTotal}
-	ResizerLoopSecondsTotal *loopSecondsTotalAdapter        = &loopSecondsTotalAdapter{metric: resizerLoopSecondsTotal}
+	ResizerSuccessResizeTotal *resizerSuccessResizeTotalAdapter = &resizerSuccessResizeTotalAdapter{metric: resizerSuccessResizeTotal}
+	ResizerFailedResizeTotal  *resizerFailedResizeTotalAdapter  = &resizerFailedResizeTotalAdapter{metric: resizerFailedResizeTotal}
+	ResizerLoopSecondsTotal   *loopSecondsTotalAdapter          = &loopSecondsTotalAdapter{metric: resizerLoopSecondsTotal}
```

Comment:
Is type of loopSecondsTotalAdapter named with different rule from resizerSuccessResizeTotalAdapter and resizerFailedResizeTotalAdapter ?